### PR TITLE
Update changelog for v0.16 release

### PR DESCRIPTION
**Pull request by ludwigschwardt**
Last Python 2 release... To be merged into `python2` branch.

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 
+0.16 (2020-08-28)
+-----------------
+* This is the last release that will support Python 2 (python2 maintenance branch)
+* New 'time_offset' sensor property that adjusts timestamps of any sensor (#307)
+* Fix calculation of cbf_dump_period for 'wide' / 'narrowN' instruments (#301)
+* Increase katstore search window by 600 seconds to find infrequent updates (#302)
+* Refactor SensorData to become a lazy abstract interface without caching (#292)
+* Refactor SensorCache to use MutableMapping (#300)
+* Fix rx_serial sensor use and file mode warning in HDF5v3 files (#298, #299)
```
Comment (bmerry) [inline NEWS.rst:12]: HDF5v3 is confusing - it's not version 3 of HDF5. I'd either use "HDF5" or "MVFv3".

**Review by bmerry — CHANGES_REQUESTED** [review]

File: NEWS.rst (hunk 1 above)
Comment (ludwigschwardt) [reply NEWS.rst]: Picked the latter - thanks!

**Review by bmerry — APPROVED** [review]
I'm happy. I've only looked at it grammatically - let me know if you'd like me to check the git diff for anything you've missed.

**Comment by ludwigschwardt** [comment]
No, it should be fine... This time I must just remember to tag the main (python2) branch and not the PR branch pre-merge as I did with v0.15 :joy:

**Pull request merged**